plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadataCompressionCodec.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import org.apache.iceberg.TableMetadataParser.Codec;
+
+public enum IcebergMetadataCompressionCodec
+{
+    NONE,
+    GZIP,
+    /**/;
+
+    public Codec toIceberg()
+    {
+        return switch (this) {
+            case NONE -> Codec.NONE;
+            case GZIP -> Codec.GZIP;
+        };
+    }
+
+    public static IcebergMetadataCompressionCodec fromIceberg(Codec format)
+    {
+        return switch (format) {
+            case NONE -> NONE;
+            case GZIP -> GZIP;
```

Comment:
should we add here a `default` option , just in case some new `Codec` value gets added in the following Iceberg versions?

Reply:
It should probably throw if it's not one of these two

Reply:
Though, do we need the Trino version of this enum, or could we just use the Iceberg one?

Reply:
I suppose we can easily miss new values if we directly use Iceberg enum.

Reply:
There is no other default than "throw" and "let it throw" is not what we want. We want to update our enum accordingly.

> Though, do we need the Trino version of this enum, or could we just use the Iceberg one?

Having our own is not a big overhead and lets us use static code analysis to know when to update the docs -- Trino docs enumerate all possible values. If we used Iceberg's class, we would want to defer to Iceberg docs for possible values.